### ISSUE TITLE
[Suggestion]: Optimizing Wither Storm Revival Mechanism

### ISSUE THREAD

**Issue by Windrinn** [nonamecrackers2/crackers-wither-storm-mod#1712]
### Description of feature request

After the Wither Storm is knocked down by the Terror Bomb, the exposed command blocks will only react to damage caused by command weapons.Hopefully, it can be added to the config file.

**Comment by TaintedBlockmObile** [comment]
It only reacts to command block tools hitting it tho. And when you go near it in survival.

**Comment by Windrinn** [comment]
> It only reacts to command block tools hitting it tho. And when you go near it in survival.

I want it not to react when approached in survival mode, unless hit with a command weapon.

**Comment by TaintedBlockmObile** [comment]
I mean like, I guess that could be some sort of config

**Comment by Windrinn** [comment]
> I mean like, I guess that could be some sort of config

Really? Thank you, I'll go check the config file.

**Comment by TaintedBlockmObile** [comment]
> > I mean like, I guess that could be some sort of config
> 
> Really? Thank you, I'll go check the config file.

No no, I mean for this suggestion. It could become a config.

**Comment by ghost** [comment]
> I want it not to react when approached in survival mode, unless hit with a command weapon.

That's impossible due to how the mod works. You don't have a command weapon by the time you use the Formidibomb, as you're supposed to get an F Bomb by phase 5. You only get command tools by phase 7, where you have to directly confront the Command Block.

I guess it could be a config like Tainted said, but I don't think it would be a good idea.

**Comment by Windrinn** [comment]
> > > I mean like, I guess that could be some sort of config
> > 
> > 
> > Really? Thank you, I'll go check the config file.
> 
> No no, I mean for this suggestion. It could become a config.

Oh, Hopefully, it can be added to the config file.

**Comment by TaintedBlockmObile** [comment]
> > I want it not to react when approached in survival mode, unless hit with a command weapon.
> 
> That's impossible due to how the mod works. You don't have a command weapon by the time you use the Formidibomb, as you're supposed to get an F Bomb by phase 5. You only get command tools by phase 7, where you have to directly confront the Command Block.
> 
> I guess it could be a config like Tainted said, but I don't think it would be a good idea.

Not necessarily, it's very possible to get a command weapon and bomb during 5.5

**Comment by Aimyx** [comment]
if you wanna wait for 3 hours you could

**Comment by TaintedBlockmObile** [comment]
Doesn't take that long lol

**Comment by WitherBean** [comment]
> ### Description of feature request
> 
> After the Wither Storm is knocked down by the Terror Bomb, the exposed command blocks will only react to damage caused by command weapons.Hopefully, it can be added to the config file.

Terror Bomb?
